Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:xenial
+
+# Apt packages
+# We don't want any interactive prompts
+RUN apt-get update && DEBIAN_FRONTEND=non-interactive apt-get install -yq doxygen graphviz patch
+
+# Set WORKDIR
+WORKDIR /documentation
```

Comment:
I probably want to use this file for builds later as well. Let's put this in `/TerminalFighter/Documentation`